src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -2078,7 +1764,6 @@
   "com.minecolonies.coremod.gui.workerhuts.school": "School",
   "com.minecolonies.coremod.gui.workerhuts.teacher": "Teacher",
   "com.minecolonies.coremod.gui.workerhuts.pupil": "Pupil",
-
```

Comment:
Now that this file isn't being stomped by datagen, it'd be nice to keep these line breaks (here and line 1757).